Add missing Node agent config sections (#24837)
newrelic/docs-website: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx

    }
    ```
  </Collapser>
  <Collapser
    id="dt-sampler-root"
    title="sampler.root"
  >
    <table>
      <tbody>
        <tr>
          <th>
            Type
          </th>

          <td>
            String or object
          </td>
        </tr>

        <tr>
          <th>
            Default
          </th>

          <td>
            `adaptive`
          </td>
        </tr>

        <tr>
          <th>
            [Environ variable](#environment)
          </th>

          <td>
            `NEW_RELIC_DISTRIBUTED_TRACING_SAMPLER_ROOT`
          </td>
        </tr>
      </tbody>
    </table>

    Controls how sampling decisions are made for root spans — transactions that do not have a remote parent (that is, no incoming `traceparent`/`newrelic` distributed tracing header). The allowable string values are:

      * `adaptive` (default): use New Relic's built-in adaptive sampling algorithm.
      * `always_on`: the agent will sample spans.
      * `always_off`: the agent will NOT sample spans.

>       * `trace_id_ratio_based`: sample a fixed ratio of traces, deterministically based on trace ID.

Reviewer: ```suggestion
    Controls how sampling decisions are made for root spans-transactions that don't have a remote parent (meaning they lack an incoming `traceparent` or `newrelic` distributed tracing header). Valid values are:

      * `adaptive` (default): Uses the built-in New Relic adaptive sampling algorithm.
      * `always_on`: Samples all spans.
      * `always_off`: Doesn't sample any spans.
      * `trace_id_ratio_based`: Samples a fixed ratio of traces, determined deterministically by the trace ID.
```

    For example, to set this in the config file, you would use:

    ```js
    distributed_tracing: {
      enabled: true,
      sampler: {
        root: 'always_on'
      }
    }
    ```

    When set to `trace_id_ratio_based`, you must also provide a `ratio` (a number between `0` and `1`):

    ```js
    distributed_tracing: {
      enabled: true,
      sampler: {
        root: {
          trace_id_ratio_based: {
            ratio: 0.5
          }
        }
      }
    }
    ```

    Or via environment variables:

    ```ini
    NEW_RELIC_DISTRIBUTED_TRACING_SAMPLER_ROOT=trace_id_ratio_based
    NEW_RELIC_DISTRIBUTED_TRACING_SAMPLER_ROOT_TRACE_ID_RATIO_BASED_RATIO=0.5
    ```

>     When set to `adaptive`, you may optionally override the sampling target (in transactions per minute, `[1, 120]`) for this sampler specifically:

Reviewer: ```suggestion
    When you set this to `adaptive`, you can optionally override the sampling target (to 1–120 transactions/minute) specifically for this sampler:
```

      </tbody>
    </table>

    When the traceparent header has a sampled flag of `01`, this controls how to handle sampling of spans. The allowable string values are:

      * `adaptive` (default): use New Relic's built-in adaptive sampling algorithm.
      * `always_on`: the agent will sample spans.
      * `always_off`: the agent will NOT sample spans.

>       * `trace_id_ratio_based`: sample a fixed ratio of traces, deterministically based on trace ID.

Reviewer: ```suggestion
      * `adaptive` (default): Uses the built-in New Relic adaptive sampling algorithm.
      * `always_on`: Samples all spans.
      * `always_off`: Doesn't sample any spans.
      * `trace_id_ratio_based`: Samples a fixed ratio of traces, determined deterministically by the trace ID.
```

      }
    }
    ```

>     When set to `trace_id_ratio_based`, you must also provide a `ratio` (a number between `0` and `1`), and when set to `adaptive` you may optionally override the sampling target for this sampler specifically — see the object-form and environment variable examples under [`sampler.root`](#dt-sampler-root) for the equivalent syntax (substitute `remote_parent_sampled` for `root`).

Reviewer: ```suggestion
    When you set this to `trace_id_ratio_based`, you must provide a `ratio` value of `0`–`1`. When you set this to `adaptive`, you can optionally override the sampling target specifically for this sampler. For the equivalent syntax, see the object-form and environment variable examples under [`sampler.root`](#dt-sampler-root) and substitute `remote_parent_sampled` for `root`.
```

      </tbody>
    </table>

    When the traceparent header has a sampled flag of `00`, this controls how to handle sampling of spans. The allowable string values are:

      * `adaptive` (default): use New Relic's built-in adaptive sampling algorithm.
      * `always_on`: the agent will sample spans.
      * `always_off`: the agent will NOT sample spans.

>       * `trace_id_ratio_based`: sample a fixed ratio of traces, deterministically based on trace ID.

Reviewer: ```suggestion
      * `adaptive` (default): Uses the built-in New Relic adaptive sampling algorithm.
      * `always_on`: Samples all spans.
      * `always_off`: Doesn't sample any spans.
      * `trace_id_ratio_based`: Samples a fixed ratio of traces, determined deterministically by the trace ID.
```

      }
    }
    ```

>     When set to `trace_id_ratio_based`, you must also provide a `ratio` (a number between `0` and `1`), and when set to `adaptive` you may optionally override the sampling target for this sampler specifically — see the object-form and environment variable examples under [`sampler.root`](#dt-sampler-root) for the equivalent syntax (substitute `remote_parent_not_sampled` for `root`).

Reviewer: ```suggestion
    When you set this to `trace_id_ratio_based`, you must provide a `ratio` value of `0`–`1`. When you set this to `adaptive`, you can optionally override the sampling target specifically for this sampler.  For the equivalent syntax, see the object-form and environment variable examples under [`sampler.root`](#dt-sampler-root) and substitute `remote_parent_not_sampled` for `root`.
```

          <td>
            `NEW_RELIC_UTILIZATION_GCP_USE_INSTANCE_AS_HOST`
          </td>
        </tr>
      </tbody>
    </table>

    When enabled, the agent uses the GCP metadata instance ID to set the hostname of the running application.
  </Collapser>

  <Collapser
    id="utilization-gcp-cloud-run-include-revision-in-host"
    title="gcp_cloud_run.include_revision_in_host"
  >
    <table>
      <tbody>
        <tr>
          <th>
            Type
          </th>

          <td>
            Boolean
          </td>
        </tr>

        <tr>
          <th>
            Default
          </th>

          <td>
            `false`
          </td>
        </tr>

        <tr>
          <th>
            [Environ variable](#environment)
          </th>

          <td>
            `NEW_RELIC_UTILIZATION_GCP_CLOUD_RUN_INCLUDE_REVISION_IN_HOST`
          </td>
        </tr>
      </tbody>
    </table>

    When enabled, and `gcp_use_instance_as_host` is also enabled, the GCP Cloud Run revision (`K_REVISION`) is prepended to the instance ID when setting the hostname of the running application, resulting in a hostname of `${K_REVISION}-${instance_id}`.
  </Collapser>
</CollapserGroup>

> ## OTLP endpoint [#otlp-endpoint]

Reviewer 2: this shouldn't be documented. In fact, we should PR the agent to remove this as it's not in use. it was the pre-cursor to hybrid agent and was being used in https://github.com/newrelic/newrelic-node-opentelemetry-integration

  </Collapser>
</CollapserGroup>

## Utilization [#utilization]

> These settings control how the agent detects information about the host and cloud environment it's running in, used to calculate utilization-based pricing. They appear in the `utilization` section of your app's `newrelic.js` configuration file.

Reviewer: ```suggestion
Configure how the agent detects host and cloud environment information to accurately calculate your utilization-based pricing. These settings appear in the `utilization` section of your app's `newrelic.js` configuration file.
```
